### ISSUE TITLE
Reordering the alt-date Widget 3 Select Boxes 

### ISSUE THREAD

**Issue by tonynd** [rjsf-team/react-jsonschema-form#297]
Not really that big of an issue, but is there any way to change the positions of the YEAR-MONTH-DAY of the alt-date widget to MONTH-DAY-YEAR?

Right now it is:
![image](https://cloud.githubusercontent.com/assets/9759347/17420149/fb6060a6-5a55-11e6-9559-a1f398441786.png)

Any help would be appreciated. Thanks!

I am using the most current version (I believe it is v0.38.0).

**Comment by mplis-jetsetter** [comment]
I don't believe there's a way to do that. You'd have to create a custom component if you wanted more control over how it looked.

**Comment by n1k0** [comment]
We could expose a dedicated widget option, though that's definitely not P1 in our case. Would anybody want to pick this up?

**Comment by shvekyha** [comment]
Hi
I would like to take this issue if possible and work on it as my first contribution:)

**Comment by heath-freenome** [comment]
@shvekyha I see your PR in your fork... are you willing to create a PR against the RJSF main branch instead?

**Comment by Soudarjya** [comment]
i would like to work on this issue as my first contri

**Comment by heath-freenome** [comment]
> i would like to work on this issue as my first contri

@Soudarjya I'm looking forward to seeing your PR. Let me know if you have any questions

**Comment by shivsperfect** [comment]
is this issue still open ?, can I make my contribution here

**Comment by heath-freenome** [comment]
it is still open

**Comment by shivsperfect** [comment]
Hi @heath-freenome seems date reorder feature is already added. 

can we close this issue.
